setup/pi/setup-teslausb

```diff
@@ -25,6 +25,7 @@ function read_setup_variables {
   newnamefor[musicsize]=MUSIC_SIZE
   newnamefor[sharename]=SHARE_NAME
   newnamefor[shareuser]=SHARE_USER
+  newnamefor[sharedomain]=SHARE_DOMAIN
```

Comment:
Remove this. This is only for backwards compatibility, and since there never was an all lowercase "sharedomain" variable, it's not needed.